### DISCUSSION TITLE
If "-H" is the "Headers" what is then "-d"

### DISCUSSION THREAD

**Discussion by StefanSladdenEng** [supabase/supabase#18035] [Questions]
Hi All, here is a somewhat stupid question from a newbie
Ive tried to find it in the Documention but without success (which probably is because of my lack of knowledge...
So here goes 
-H "Content-Type: application/json" \
-H "Prefer: return=minimal" \
**-d** '{ "some_column": "someValue", "other_column": "otherValue" }'  

Where do i put **-d** variables - I guess it is not a Header :-D ? And if not - What is it?
Need to know to be able to apply the call the right way... 
Hoping for a quick reply.. 
Cheers

**Comment by GaryAustin1** [comment]
Not completely clear what you are doing.  You seem to be looking at the Bash scripting.  I assume you are trying to do your own REST API calls versus use a supabase client.  The value after -d is the data body for a request like POST that has a body.
https://reqbin.com/req/curl/c-d2nzjn3z/curl-post-body

**Reply by StefanSladdenEng** [reply]
Aha - This is what they are writing 
INSERT A ROW
curl -X POST 'https://myprojectcode.supabase.co/rest/v1/crossdatabas' \
-H "apikey: SUPABASE_KEY" \
-H "Authorization: Bearer SUPABASE_KEY" \
-H "Content-Type: application/json" \
-H "Prefer: return=minimal" \
**-d** '{ "some_column": "someValue", "other_column": "otherValue" }'

If i use "Body" instead of "Header" for this one it should work i guess....
Ill try that and let you know... 

Thanks for now...
